### ISSUE TITLE
make in tarball made with make distrib is not working

### ISSUE THREAD

**Issue by matkoniecz** [mnemosyne-proj/mnemosyne#73]
```
mateusz@grisznak:~/Install/mnemosyne/dist/Mnemosyne-2.6.1$ make
# Just the bare minimum to get things running
cd mnemosyne/pyqt_ui && make
make[1]: Entering directory '/home/mateusz/Install/mnemosyne/dist/Mnemosyne-2.6.1/mnemosyne/pyqt_ui'
make[1]: *** No targets specified and no makefile found.  Stop.
make[1]: Leaving directory '/home/mateusz/Install/mnemosyne/dist/Mnemosyne-2.6.1/mnemosyne/pyqt_ui'
makefile:35: recipe for target 'build' failed
make: *** [build] Error 2
```

**Comment by pbienst** [comment]
Try 'make build-all-deps' first. Does that work? The new github instructions seem to have missed that part.

**Comment by matkoniecz** [comment]
Thanks for the help!

I now tried:

```
make build-all-deps
sudo make distrib
cd dist
sudo tar -zxvf "Mnemosyne-2.6.1.tar.gz"
cd Mnemosyne-2.6.1
make
```

still failed with

```
mateusz@grisznak:~/Install/mnemosyne/dist/Mnemosyne-2.6.1$ make
# Just the bare minimum to get things running
cd mnemosyne/pyqt_ui && make
make[1]: Entering directory '/home/mateusz/Install/mnemosyne/dist/Mnemosyne-2.6.1/mnemosyne/pyqt_ui'
make[1]: *** No targets specified and no makefile found.  Stop.
make[1]: Leaving directory '/home/mateusz/Install/mnemosyne/dist/Mnemosyne-2.6.1/mnemosyne/pyqt_ui'
makefile:35: recipe for target 'build' failed
make: *** [build] Error 2
```
```

**Comment by pbienst** [comment]
If you're running from within the tarball, installing should proceed by 'sudo python setup.py install'. It seems in the move to github, the documentation really got messed up...

If you're running from within the checked-out source, 'make build-all-deps' followed by 'make' should allow you to run Mnemosyne without installing it to the system directories. (It will actually start up Mnemosyne using a debug datadir)

**Comment by matkoniecz** [comment]
> If you're running from within the tarball, installing should proceed by 'sudo python setup.py install'.

Then why tarball includes makefile? It strongly suggests using standard `make` command.

`sudo python setup.py install` from generated unpacked tarball circles back to #69

```
Traceback (most recent call last):
  File "/usr/local/bin/mnemosyne", line 4, in <module>
    __import__('pkg_resources').run_script('Mnemosyne==2.6.1', 'mnemosyne')
  File "/home/mateusz/.local/lib/python3.5/site-packages/pkg_resources/__init__.py", line 654, in run_script
    self.require(requires)[0].run_script(script_name, ns)
  File "/home/mateusz/.local/lib/python3.5/site-packages/pkg_resources/__init__.py", line 1425, in run_script
    .format(**locals()),
pkg_resources.ResolutionError: Script 'scripts/mnemosyne' not found in metadata at '/home/mateusz/Install/mnemosyne/dist/Mnemosyne-2.6.1/Mnemosyne.egg-info'
```

>  'make build-all-deps' followed by 'make' 

I tried that

```
mateusz@grisznak:~/Install/mnemosyne$ make
# For debugging: running the code in place.
PYTHONPATH=. python3 mnemosyne/pyqt_ui/mnemosyne -d dot_mnemosyne2
Warning: Could not import OpenGL. Might cause a black screen on some Linux distributions. Try installing python3-opengl in that case.
Log body:
 An unexpected error has occurred.
Please forward the following info to the developers:

Traceback (innermost last):
  File "mnemosyne/pyqt_ui/mnemosyne", line 145, in <module>
    from PyQt5 import QtWebEngineWidgets  # Needs to happen first for some reason.
 ImportError: cannot import name 'QtWebEngineWidgets'

An unexpected error has occurred.
Please forward the following info to the developers:

Traceback (innermost last):
  File "mnemosyne/pyqt_ui/mnemosyne", line 145, in <module>
    from PyQt5 import QtWebEngineWidgets  # Needs to happen first for some reason.
 ImportError: cannot import name 'QtWebEngineWidgets'
An unexpected error has occurred.
Please forward the following info to the developers:

Traceback (innermost last):
  File "mnemosyne/pyqt_ui/mnemosyne", line 145, in <module>
    from PyQt5 import QtWebEngineWidgets  # Needs to happen first for some reason.
 ImportError: cannot import name 'QtWebEngineWidgets'
makefile:31: recipe for target 'run' failed
make: *** [run] Error 1
```

`python3-opengl` was uninstalled but installing it and rerunning 

```
make build-all-deps
make
```

failed to change anything.

**Comment by pbienst** [comment]
>Then why tarball includes makefile? It strongly suggests using standard make command.
Correct, fixing this now. (Before the move to github, there was a separate file with the Linux build instructions, but this got merged into the github README.md and was lost.)

>Script 'scripts/mnemosyne' not found in metadata 
The only references I find to errors like this are related to a broken install in some way. I'm afraid I will need to rely on you for this to figure out what exactly is going on, since I don't have a linux box anymore...

>ImportError: cannot import name 'QtWebEngineWidgets'
You need to make sure this is installed on your system. Hopefully, this is the final hurdle :-)

**Comment by matkoniecz** [comment]
I now followed a new better README and it solved problem that I reported here. Thanks!
